src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -168,8 +168,16 @@ private JObject InitializeHostConfig(string hostJsonPath, JObject hostConfigObje
                         TryWriteHostJson(hostJsonPath, hostConfigObject);
                     }
 
-                    if (hostConfigObject["version"]?.Value<string>() != "2.0")
+                    string hostJsonVersion = hostConfigObject["version"]?.Value<string>();
```

Comment:
do we check somewhere if version property is present?

Reply:
Yeah I think the "?" takes care of that, right?